solidity/random-beacon/package.json

```diff
@@ -16,7 +16,7 @@
   "scripts": {
     "clean": "hardhat clean && rm -rf cache",
     "build": "hardhat compile",
-    "test": "hardhat check-accounts-count && hardhat test",
+    "test": "TEST_USE_STUBS_BEACON=true hardhat check-accounts-count && hardhat test",
```

Comment:
```suggestion
    "test": "hardhat check-accounts-count && TEST_USE_STUBS_BEACON=true hardhat test",
```

Reply:
How about we unify the namings when we use contract stubs? What I mean is that in the `tags` we have `allowStubs`, here we have `..USE_STUBS_..`. WDYT if we have something like `tags: ["contractStubs"]` and `"test": "CONTRACT_STUBS_BEACON=true hardhat test" / "CONTRACT_STUBS_ECDSA=true hardhat test"`?

Reply:
To be honest I like the current solution better as the names explain pretty well what they mean.